### DISCUSSION TITLE
Git action publishes on GitHub packages, but not possible to publish on nuget.org

### DISCUSSION THREAD

**Discussion by sonomirco** [NuGet/Home#11104] [Q&A] [answered]
Hi there,
I am testing this action here.
https://github.com/sonomirco/Pipeline/packages/919112?version=1.1.11

As you can see, the action is able to publish the package to GitHub packages but when it tries to publish on nuget.org it says the package is already existing.
How is this possible?

If I search the package on NuGet I haven't any results.
![image](https://user-images.githubusercontent.com/57708659/127463203-f82a4068-7ff7-4d44-b70f-9bb2191d4819.png)
if I try to upload the package manually, this is what I got.
![image](https://user-images.githubusercontent.com/57708659/127463342-77884a21-070c-4b1e-b1fd-f72aee590e46.png)

I can't find the problem.
Thank you in advance for the help!

**Comment by joelverhagen** [comment, marked as answer]
Hey @sonomirco, not all ID prefix reservations are discoverable on the search results. Today, there is no way to enumerate all of the namespaces that are reserved. In this particular case, `Pipeline.*` is reserved. If you would like to go through the application process for getting access to this namespace or some part of it, feel free to go through the application process documented here:
https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation#id-prefix-reservation-application-process

However, I would recommend considering a less generic prefix than "Pipeline". One common pattern is putting your organization or commonly used username at the beginning of the package ID. For example, Microsoft package start with "Microsoft". My own personal packages start with "Knapcode" which is the name I use my coding projects 😃. It gives me leeway to have relatively generic words as the second part of the package ID/namespace and avoid conflicts or ambiguity with other package authors.
